content/blog/grants-and-language-update/contents.lr

```diff
@@ -0,0 +1,52 @@
+title: Extended grant support, ORCID and language field.
+---
+author: knowak
+---
+body:
+Today, we are introducing three additions to Zenodo:
+
+1. Extended grant support
+2. Language field
+3. ORCID for authors
+
+## Extended grant support (powered by OpenAIRE)
+
+We are expanding our grants database with over 620,000 grants from 8 new funders such as National Science Foundation (US) and Wellcome Trust (UK) - all thanks to [OpenAIRE](https://openaire.eu/)'s ever growing grants database.
+
+![Zenodo OpenAIRE grants database](grants.png)
+
+So far we have been supporting only grants from the European Commission (FP7 and Horizon 2020). Today our dataset contains grants from the following funders:
```

Comment:
So far we have **only** been supporting ~~only~~ grants
Today our dataset **has grown and** contains grants